tests/test_model.py

```diff
@@ -43,7 +43,15 @@ def test_against_gpt_neox_model(rotary_pct, batch_size, n_embd, parallel_residua
     from scripts.convert_hf_checkpoint import copy_weights_gpt_neox
 
     batch_size = 3
-    ours_config = Config(block_size=64, vocab_size=100, n_layer=4, n_head=8, n_embd=n_embd)
+    ours_config = Config(
```

Comment:
Before `rotary_pct` and `parallel_residual` arguments were provided, but not used.

Reply:
This is also fixed in https://github.com/Lightning-AI/lit-gpt/pull/606

Reply:
Oh, I see.
Should I then remove these changes from my PR, since yours is technically was first to change it? 😄

Reply:
No, it's okay. I just wanted to mention that this wasn't forgotten